src/process.c

```diff
@@ -226,9 +226,7 @@ cc_oci_setup_shim (struct cc_oci_config *config,
 	ret = true;
 
 out:
-	if (tty_fd > 2) {
```

Comment:
/me admits it rang a little bell when he read it... iirc, this was so we do not close the host stdin/stdout/stderr iirc - and if so, I will nag that there was not enough comments in either the commit message nor the code... @amshinde - can you remember ? (and if so, maybe submit a new PR clarifying in the code etc. etc.)